db_scripts/dump.sh

```diff
@@ -1,6 +1,17 @@
 #!/bin/bash
 
-# For AMI: sudo yum -y install https://download.postgresql.org/pub/repos/yum/9.6/redhat/rhel-6-x86_64/pgdg-ami201503-96-9.6-2.noarch.rpm
-#          sudo yum -y install postgresql96
+# export ENV_NAME=prod
+# aws cloudformation create-stack --region us-east-1 --stack-name $ENV_NAME-bastionhosts --template-url https://s3.amazonaws.com/crowd-deployment/infrastructure/bastion-hosts.yaml --parameters ParameterKey=EnvironmentName,ParameterValue=$ENV_NAME ParameterKey=KeyPairName,ParameterValue=rstorey@loc.gov --disable-rollback
+# aws cloudformation delete-stack --region us-east-1 --stack-name $ENV_NAME-bastionhosts
 
-pg_dump -Fc --clean --create --no-owner --no-acl -U concordia -h $POSTGRESQL_HOST concordia -f concordia.dmp
\ No newline at end of file
+export TODAY=20190129
```

Comment:
We could make the date dynamic:

```suggestion
export TODAY=$(date +%Y%m%d)
```

Reply:
Thanks for this, saved me some time looking up the date format documentation